src/cookies.js

```diff
@@ -100,14 +100,15 @@ export function setCookie(win, name, value, expirationTime, opt_options) {
   if (opt_options && opt_options.domain) {
     domain = opt_options.domain;
   } else if (opt_options && opt_options.highestAvailableDomain) {
-    domain = getHighestAvailableDomain(win);
+    domain = /** @type {string} */ (getHighestAvailableDomain(win));
```

Comment:
Why are we typecasting here? It can be `null`, too.

Reply:
```js
Type checking failed:
src/cookies.js:105: ERROR - [JSC_TYPE_MISMATCH] actual parameter 5 of trySetCookie$$module$src$cookies does not match formal parameter
found   : (null|string|undefined)
required: (string|undefined)
  trySetCookie(win, name, value, expirationTime, domain);
                                                 ^^^^^^
```